aten/src/ATen/cpu/vec/vec512/vec512_int.h

```diff
@@ -1002,13 +1002,37 @@ Vectorized<T> inline int_elementwise_binary_512(const Vectorized<T>& a, const Ve
 template <>
 Vectorized<int8_t> inline operator*(const Vectorized<int8_t>& a, const Vectorized<int8_t>& b) {
   // We don't have an instruction for multiplying int8_t
```

Comment:
May we remove the comment `We don't have an instruction for multiplying int8_t`? Thanks!